requirements.txt

```diff
@@ -37,11 +37,11 @@ idna==2.7                 # via -r requirements.in, requests, tldextract, yarl
 importlib-metadata==0.23  # via -r requirements.in, pluggy, pre-commit, pytest, virtualenv
 jinja2==2.11.2            # via -r requirements.in, cnx-epub
 jwcrypto==0.6.0           # via -r requirements.in, github3.py
-lxml==4.5.0               # via -r requirements.in, cnx-epub, cnx-litezip, cnxml, rhaptos.cnxmlutils
+lxml==4.4.3               # via -r requirements.in, cnx-epub, cnx-litezip, cnxml, rhaptos.cnxmlutils
 markupsafe==1.1.1         # via -r requirements.in, jinja2
 more-itertools==8.2.0     # via -r requirements.in, pytest
 multidict==4.7.5          # via aiohttp, yarl
-nebuchadnezzar==9.6.0     # via -r requirements.in
+git+https://github.com/openstax/nebuchadnezzar.git
```

Comment:
I'm not sure if these changes are intentional for this PR or an accidental inclusion, but in general I believe the `requirements.txt` file is [auto-generated in this repo](https://github.com/openstax/cnx-automation#doing-updates-to-dependencies). If there are proposed dependency version changes that need to be made (?), they should be reflected in a patch to `requirements.in`.